packages/@expressive-code/plugin-collapsible-sections/src/styles.ts

```diff
@@ -0,0 +1,78 @@
+import { StyleSettings, ExpressiveCodeTheme, ResolvedCoreStyles } from '@expressive-code/core'
+
+export const collapsibleSectionClass = 'ec-section'
+
+export const collapsibleSectionsStyleSettings = new StyleSettings({
+	closedBorderWidth: '0px',
+	closedPadding: '4px 0 4px var(--padding-inline)',
+	closedMargin: '0rem',
+	closedTextColor: '',
+	closedBackgroundColor: 'rgb(84 174 255 / 20%)',
+	closedBorderColor: 'rgb(84 174 255 / 50%)',
+	openBorderWidth: '1px',
+	openPadding: '0rem',
+	openMargin: '0rem',
+	openBackgroundColor: 'rgb(246 248 250 / 7.5%)',
+	openBorderColor: 'rgb(216 222 228 / 10%)',
+})
+
```

Comment:
```suggestion
import { StyleSettings, ExpressiveCodeTheme, ResolvedCoreStyles, setAlpha } from '@expressive-code/core'

export const collapsibleSectionClass = 'ec-section'

export const collapsibleSectionsStyleSettings = new StyleSettings({
	closedBorderWidth: '0',
	closedPadding: '4px 0 4px var(--padding-inline)',
	closedMargin: '0',
	closedFontFamily: 'inherit',
	closedFontSize: 'inherit',
	closedLineHeight: 'inherit',
	closedTextColor: 'inherit',
	closedBackgroundColor: ({ theme }) => setAlpha(theme.colors['editor.foldBackground'], 0.2) || 'rgb(84 174 255 / 20%)',
	closedBorderColor: ({ theme }) => setAlpha(theme.colors['editor.foldBackground'], 0.5) || 'rgb(84 174 255 / 50%)',
	openBorderWidth: '1px',
	openPadding: '0',
	openMargin: '0',
	openBackgroundColor: 'transparent',
	openBorderColor: 'transparent',
})

```

I'd recommend making the font configurable, because users may want to use the UI font rather than the code font here. I've tried both and can't decide which is better for this use case.

The colors should be derived from the theme (if any are defined in there) and fall back to your defaults if there are none.

Also, I'd like the open details background to be the same as the default background as they no longer provide any functionality after being expanded, so I'd set these to `transparent` by default.